frontend/src/index.tsx

```diff
@@ -1,6 +1,5 @@
-import { createRoot } from 'react-dom/client';
+import { render } from 'react-dom';
 import { App } from './App';
 
 const container = document.getElementById('root');
-const root = createRoot(container);
-root.render(<App />);
+render(<App />, container);
```

Comment:
Rolling back concurrent rendering here is a workaround to avoid https://github.com/visgl/react-map-gl/issues/1960, which crashes the map when it mounts.